src/utils/daos_hdlr.c

```diff
@@ -3074,7 +3043,7 @@ dm_disconnect(struct cmd_args_s *ap,
 			rc = dfs_umount(src_file_dfs->dfs);
 			if (rc != 0) {
 				fprintf(ap->errstream, "failed to unmount source (%d)\n", rc);
```

Comment:
```suggestion
				DH_PERROR_SYS(ap, rc, "failed to unmount source", rc);
```

Reply:
There are many, many, places in this file I could have updated but didn't, I've spent most of the day on this and probably only have about a third.  In the end I decided to write the output filter, make sure the ones the test covers are correct and leave it to @dsikich to pick up the rest.

Reply:
@daltonbohning @ashleypittman yeah, I expect to pick up anything not covered by this PR. Updating to the dfs_sys API will also help, and I plan to start that this week or next.